src/Symfony/Component/Serializer/Normalizer/FormErrorNormalizer.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\Form\FormInterface;
+
+/**
+ * Normalizes invalid Form instances.
+ */
+class FormErrorNormalizer implements NormalizerInterface, CacheableSupportsMethodInterface
+{
+    const TITLE = 'title';
+    const TYPE = 'type';
+    const CODE = 'status_code';
```

Comment:
I don't think that these constants are needed.

Reply:
I was inspired from the `ConstraintViolationListNormalizer`.